src/main/java/teammates/ui/controller/ImageUploadAction.java

```diff
@@ -0,0 +1,99 @@
+package teammates.ui.controller;
+
+import java.util.List;
+import java.util.Map;
+
+import teammates.common.util.Config;
+import teammates.common.util.Const;
+
+import com.google.appengine.api.blobstore.BlobInfo;
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+
+public class ImageUploadAction extends Action {
+
+    FileUploadPageData data;
+
+    @Override
+    protected ActionResult execute() {
```

Comment:
Looks like this method is not removed in `AdminEmailImageUploadAction`, which causes duplication.